### PR TITLE
Template Management Branch

### PR DESCRIPTION
**Pull request by satrams**
The skeletons of a system to manage templates inside of the app, including base systems for creating a template and removing all templates for development purposes.

Also includes a markdown file detailing upcoming project of authentication architecture.

### CODE REVIEW

File: read-docs/start.sh [hunk 1]
```diff
@@ -5,4 +5,4 @@ set -a
 . $DIR/../.env
 set +a
 
-npx babel-watch ${DIR}/index.js
```
Comment (ryan-mahoney) [inline read-docs/start.sh]: @satrams was this change intentional?

**Review by ryan-mahoney — APPROVED** [review]
Excellent PR.

Left one small comment.

File: read-docs/start.sh (hunk 1 above)
Comment (satrams) [reply read-docs/start.sh]: Haha no I must've forgotten to change it back
